core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -216,6 +216,7 @@ object TestUtils extends Logging {
                          saslSslPort: Int = RandomPort,
                          rack: Option[String] = None,
                          logDirCount: Int = 1,
+                         replicaFetchBackoff: Int = 1000,
```

Comment:
This is not common enough to be here. I suggest setting the config on the returned `Properties` instance.

Reply:
That's fair. updated.